students/jiyeon/users/views.py

```diff
@@ -53,12 +56,62 @@ def post(self, request):
                 ,status=400)
 
         if not User.objects.filter(email=data['email']).exists():
+            password        = data['password'].encode('utf-8')
+            hashed_password = bcrypt.hashpw(password, bcrypt.gensalt())
+
             User.objects.create(
                 name          = data['name'],
                 email         = data['email'],
-                password      = data['password'],
+                password      = hashed_password.decode('utf-8'),
                 phone_number  = data['phone_number']
             )
             return JsonResponse({"message" : f"사용자 등록 성공, {data['email']}"}, status=201)
 
-        return JsonResponse({"E-mail Error": "이미 가입된 회원입니다."}, status=401)
\ No newline at end of file
+        return JsonResponse({"E-mail Error": "이미 가입된 회원입니다."}, status=401)
+
+class LoginView(View):
+    def post(self, request):
+        # POST 127.0.0.1:8000/login
+        """
+        1. 목적 : 클라이언트로부터 받은 사용자 데이터(이메일, 비밀번호)를 데이터베이스와 대조하여
+                 인증하는 용도
+        2. INPUT : 
+        {
+            "email"        : "jiyeon@email.com",
+            "password"     : "jiyeon_KIM_pw^^"
+        }
+        3. OUTPUT : 
+        성공 시
+        {
+            "message" : "사용자 로그인 성공, jiyeon@email.com", status=200
+        }
+        실패 시
+        {
+            "message" : 사용자 로그인 실패, 이메일 혹은 비밀번호를 확인해주세요.", status=401
+        }
+        """
+        data = json.loads(request.body)
+
+        if 'email' not in data:
+
```

Comment:
if문과 return 문 사이에 한칸씩 띄워져있는데 지워주시면 나머지는 👍🏻 
```suggestion
        if 'email' not in data:
```